code/modules/client/client_procs.dm

```diff
@@ -292,13 +293,23 @@ GLOBAL_LIST(external_rsc_urls)
 	else if (isnum(player_age) && player_age < config.notify_new_player_age)
 		message_admins("New user: [key_name_admin(src)] just connected with an age of [player_age] day[(player_age==1?"":"s")]")
 
-	if(!IsGuestKey(key) && SSdbcore.IsConnected())
-		findJoinDate()
-
 	sync_client_with_db(tdata)
 	get_message_output("watchlist entry", ckey)
 	check_ip_intel()
 
+	//Use the byond account age for jobs, instead of time on the server itself if it was found
+	//otherwise fall back to the player age which should be less anyway
+	if(config.use_account_age_for_jobs && byond_age >= 0)
+		player_age = byond_age
+
+	if(byond_age > 0 && byond_age < config.notify_new_player_byond_age)
```

Comment:
Since you're checking if the age is greater than 0, it's going to give an error message for accounts created on the same day as first connection.